.github/ISSUE_TEMPLATE/kubernetes_bump.md

```diff
@@ -12,6 +12,7 @@ This issue is tracking the tasks that should be implemented **after** the Kubern
 ## Tasks
 
 Prerequisites:
+
```

Comment:
are there supposed to be prerequisites here like we have below (a bullet list)?

it appears like the prerequisites are a checklist, is that accurate?

Reply:
Removed the first point as suggested by stefan and moved the other to a note :-)